format.go

```diff
@@ -9,6 +9,7 @@ import (
 	cid "github.com/ipfs/go-cid"
 )
 
+// Resolver is the interface that details paths a node contains
```

Comment:
`Resolver` is more something that can be resolved through than something that "details" (although I'm planning on reworking these interfaces in the near future).